### PR TITLE
make accounting tests fast

### PR DESCRIPTION
**Pull request by nickpell**

### CODE REVIEW

**Comment by emord** [comment]
@nickpell any reason to not merge this asap?

File: corehq/apps/analytics/tests/test_subscription_properties.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def _setup_subscription(cls, domain_name, software_plan):
         cls._to_delete.append(subscription)
 
     @classmethod
-    def tesrDownClass(cls):
+    def tearDownClass(cls):
```
Comment (benrudolph) [inline corehq/apps/analytics/tests/test_subscription_properties.py]: lol

**Comment by benrudolph** [comment]
heroic!

**Comment by nickpell** [comment]
@emord nope, tests are passing so looks ready to go!

File: corehq/apps/analytics/tests/test_subscription_properties.py (hunk 1 above)
Comment (czue) [inline corehq/apps/analytics/tests/test_subscription_properties.py]: lsl

**Pull request merged**